cmd/ocm/gcp/gcp.go

```diff
@@ -0,0 +1,34 @@
+package gcp
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	TargetDir                string
+	Region                   string
+	Name                     string
+	Project                  string
+	WorkloadIdentityPool     string
+	WorkloadIdentityProvider string
+	DryRun                   bool
+}
+
+// NewGCPCmd implements the "gcp" subcommand for the credentials provisioning
+func NewGCPCmd() *cobra.Command {
+	gcpCmd := &cobra.Command{
+		Use:   "gcp COMMAND",
+		Short: "Perform actions related to GCP WIF",
+		Long:  "Manage GCP Workload Identity Federation (WIF) resources.",
+		Args:  cobra.MinimumNArgs(1),
+	}
+
+	gcpCmd.AddCommand(NewCreateWorkloadIdentityConfiguration())
```

Comment:
@JakobGray  The convention for cli cmd syntax is as below

ocm gcp create  <resource>.  i.e ocm gcp create wif_config 
same for delete , list  etc.

Please change it accordingly